lgc/patch/FragColorExport.cpp

```diff
@@ -99,224 +99,170 @@ namespace lgc {
 FragColorExport::FragColorExport(LLVMContext *context) : m_context(context) {
 }
 
+// =====================================================================================================================
+// Split a possible vector value, if there are less than 4 elements, append with null value.
+//
+// @param output : Fragment color output
+// @param builder : The IR builder for inserting instructions
+// @param [out] results : the returned elements
+static void splitVector(Value *input, BuilderBase &builder, Value *results[4]) {
```

Comment:
Better to return the results through a `SmallVectorImpl` or `MutableArrayRef`

Reply:
sounds good.